CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Correct log messages in ERC20-BTC example.
 
 ### Added
+- Added a simple negotiation protocol prototype to the example in `seperate_apps`.
```

Comment:
```
- Added a simple negotiation protocol prototype to the example in `separate_apps`.
```

Reply:
Please keep this in mind when working on future changes as this PR is already merged :)